script/openqa-clone-job

```diff
@@ -141,7 +144,7 @@ my $jobid;
 
 sub usage($) {
     my $r = shift;
-    eval { require Pod::Usage; Pod::Usage::pod2usage($r); };
+    eval { require Pod::Usage; Pod::Usage::pod2usage(-exitval => $r, -verbose => 99) };
```

Comment:
interesting. What is the meaning of this "verbose => 99"?

Reply:
It causes `Pod::Usage::pod2usage` to display all the sections. Maybe the duplication of SYNOPSIS was previously done to workaround the default behavior (to answer the question from your other comment).

Reply:
I see. Could be. all good then.